docker-for-mac/edge-release-notes.md

```diff
@@ -12,6 +12,21 @@ For information about Stable releases, see the [Stable release
 notes](release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.2.0
```

Comment:
Should this say Edge?

Reply:
This has always been Community apparently, they are different pages all together for Stable and Edge.

Reply:
Yes, we have specific pages for Stable and Edge release notes and both pages state they are the 'Community' releases.